### ISSUE TITLE
Offline Version frame not working

### ISSUE THREAD

**Issue by undeadf0x** [chinchang/web-maker#563]
## Meta info

- Is this a bug or suggestion?: Bug
- Version: 6.3.1
- Context: Web app

Since the Web Maker webapp page moved from `/app/` to `/create/`, it appears that the offline version's frame hasn't saved properly
![image](https://github.com/chinchang/web-maker/assets/62490883/771ce71b-6bfa-4e1e-b6c6-09575d634112)

**Comment by chinchang** [comment]
Yes, the security feature introduced on the iframe (now capable of running public creations) seems to have interfered with the offline feature. I am on it.

**Comment by labbo-lab** [comment]
Hello, any update on this? It still appears to not be working

**Comment by chinchang** [comment]
This is fixed now. The Web Maker web app is back offline!
